### ISSUE TITLE
`go list` need at least one `.go` file in the root of the project 

### ISSUE THREAD

**Issue by xmlking** [markbates/pkger#34]
apparently  `go list` won't work if there is no `.go` files in the root of the project. example https://github.com/xmlking/micro-starter-kit

root Issue: https://github.com/golang/go/issues/33855

when I try `pkger list` it is returning no results. 

```bash
pkger list -include /srv             
2019/11/13 11:37:25 exit status 1: "go list -json -find github.com/xmlking/micro-starter-kit": can't load package: package github.com/xmlking/micro-starter-kit: cannot find module providing package github.com/xmlking/micro-starter-kit
```
I am adding a dummy `doc.go` in the project root as a workaround. 

we may need to document this...

**Comment by markbates** [comment]
This should be fixed in `v0.12.3`.

**Comment by s12chung** [comment]
doesn't seem to be fixed in v0.15.1
